kola/harness.go

```diff
@@ -358,7 +358,20 @@ func getClusterSemver(pltfrm, outputDir string) (*semver.Version, error) {
 		return nil, fmt.Errorf("parsing /etc/os-release: %v: %s", err, stderr)
 	}
 
-	version, err := semver.NewVersion(strings.Split(string(out), "=")[1])
+	// TODO: add distro specific version handling
+	switch distro {
+	case "cl":
+		return parseCLVersion(out)
+	case "rhcos":
```

Comment:
I think these should be seperate cases. `default` should error and rhcos should return the empty semver

Reply:
Dropping the `default` case and changing the end of function return to:

```golang
return nil, fmt.Errorf("no case to handle distribution %q", distro)
```